src/styles/settings.scss

```diff
@@ -17,12 +17,27 @@
   width: 150px;
 }
 
-.color-variant {
+.color-variant-preview {
```

Comment:
@balanejacob You can put the styles directly on the component file inside a `style` tag. In Vue it's usually better to put the styles directly on the component so you can see them right away when you open the file.

Global styles that apply to the whole app can be placed in a separate file.